CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Change Log for AWS IoT Device Shadow
```

Comment:
Changed the title to the official title

Reply:
Isn't this name different from one in the README?